lib.py

```diff
@@ -152,30 +148,17 @@ async def build_code(author_name: str, author_id: int, tmp_dir: str) -> bool:
         return False
 
 
-def load_answers(cursor: Cursor, day: int, part: int):
-    """Load the expected answers for each input file."""
-    rows = cursor.execute(
-        "SELECT key, answer2 FROM solutions WHERE day = ? AND part = ?",
-        (day, part),
-    )
-
-    answer_map = {}
-    for r in rows:
-        (key, answer) = r
-        answer_map[key] = answer
-
-    return answer_map
-
-
-def get_input_files(day: int) -> list[str]:
+def get_input_files(day: int) -> list[SessionLabel]:
     """
     List all the input files that exist for the current day.
     """
     day_path = get_input_dir_for_day(day)
-    return [f for f in os.listdir(day_path) if os.path.isfile(os.path.join(day_path, f))]
+    return [
+        SessionLabel(f) for f in os.listdir(day_path) if os.path.isfile(os.path.join(day_path, f))
```

Comment:
The existing code works by storing a file name in the DB, not a Session Label as the config file would understand it.

Does that change how you think about the types here?

Reply:
tbh i wanted to move this function into the database api so i didn't think too much about it yet
SessionLabel is meant to the the config files session label yes, so this code is being bad

Reply:
If you update the fetch script to write to the DB instead of the filesystem, then it's fine to move this into the DB, and everything _should_ work out correctly.